### PR TITLE
fixes #15366 - add yum-plugin-security dep to el5/6 packages

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: katello-agent/katello-agent.spec [hunk 1]
```diff
@@ -24,6 +24,9 @@ Requires: python-gofer-proton >= 2.5
 Requires: python-pulp-agent-lib >= 2.6
 Requires: pulp-rpm-handlers >= 2.6
 Requires: subscription-manager
+%if 0%{?rhel} < 6
```
Comment (ehelms) [inline katello-agent/katello-agent.spec]: This would only install it on EL5.

Comment (komidore64) [inline katello-agent/katello-agent.spec]: doh! that should be a `7`!

**Comment by ehelms** [comment]
Can you give a brief comment on why this fixes the described issue?

**Comment by komidore64** [comment]
This spec change isn't actually the fix. The fix is in `pulp-2.8.4` and related packages. Also requires `yum-plugin-security` to be included explicitly on RHEL versions older than `7` because the yum on those OSes hadn't slurped in `yum-plugin-security` into core yet.

**Comment by ehelms** [comment]
ACK - sorry for the long delay!

**Comment by komidore64** [comment]
:+1:

**Pull request merged**